components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -205,10 +204,17 @@ ConvertGreaselionRuleToExtensionOnTaskRunner(
             extensions::file_util::GetComputedHashesPath(extension->path()));
   }
 
-  // Take ownership of this temporary directory so it's deleted when
-  // the service exits
-  return std::make_pair(extension, std::move(temp_dir));
+  // Take ownership of this temporary directory; the service will delete it on
+  // exit
+  return std::make_pair(extension, temp_dir.Take());
 }
+
+void DeleteExtensionDirs(const std::vector<base::FilePath>& extension_dirs) {
```

Comment:
this should be in an anonymous namespace at the top of the file

Reply:
Not sure I follow... This is already inside an anonymous namespace, and such namespace is at the top of the file. Or do you mean that you'd like to have this particular function at the top of the namespace as well, as the first function in it?

Reply:
We chatted over Slack last night; this comment was just a misunderstanding and we can ignore it.